lib/install/management/create.go

```diff
@@ -45,6 +45,17 @@ func (d *Dispatcher) CreateVCH(conf *config.VirtualContainerHostConfigSpec, sett
 
 	var err error
 
+	// resource pool path determined based on DRS setting.  If DRS is disabled then
+	// no resource pools are supported, so use the provided compute path.  If enabled then
+	// append the appliance name to the path and a pool will be created.
+	if conf.DRSDisabled {
+		d.vchPoolPath = settings.ResourcePoolPath
+	} else {
+		d.vchPoolPath = path.Join(settings.ResourcePoolPath, conf.Name)
+	}
+
+	// TODO:  This check's existence via resourcePool -- with the intro of inventory
```

Comment:
As long as we get the correct resource pool we should still be able to determine if one of the VM's is the VCH right? So this should still be good I think. The inventory entry for the vm will be moved, but should still be in the same resource pool with my changes.